docs/spec/auth/scope.md

```diff
@@ -78,7 +88,8 @@ scopes.
 ```
 scope                   := resourcescope [ ' ' resourcescope ]*
 resourcescope           := resourcetype  ":" resourcename  ":" action [ ',' action ]*
-resourcetype            := /[a-z]*/
+resourcetype            := resourcetypevalue [ '(' resourcetypevalue ')' ]
+resourcetypevalue       := /[a-z]+/
```

Comment:
Should `resourcetypevalue` be `/[a-z0-9]+/` for agreement with `typeRegexp` in contrib/token-server/token.go?